demos/tests/run_tests.py

```diff
@@ -202,6 +219,14 @@ def option_to_args(key, value):
                 print()
                 device_args = demo.device_args(args.devices.split())
                 for test_case_index, test_case in enumerate(demo.test_cases):
+                    def get_models(case, keys):
+                        models = []
+                        for key in keys:
+                            model = case.options.get(key, None)
+                            if model:
+                                models.append(model.name if isinstance(model, ModelArg) else model.model_name)
+                        return models
+                    test_case_models = get_models(test_case, demo.model_keys)
```

Comment:
Do we really need this written as separate local function ?  Is it called anywhere more than once ? I'd suggest either write it as separate global function, or local function, but at the very beginning of main (definetely not inside the cycle), or just put its code inside the cycle without creating extra function.

Reply:
Done